setup.py

```diff
@@ -59,7 +59,8 @@ def get_packages(package):
         'PyYAML>=3.10',
         'tornado>=5.0',
         'ghp-import>=1.0',
-        'pyyaml_env_tag>=0.1'
+        'pyyaml_env_tag>=0.1',
+        'importlib_metadata>=3.10'
```

Comment:
Could this be specified as required only for Python<3.8  (which it is)?

Reply:
No, the API changes in Python 3.10 and we are using the new API. Therefore, we need the backport for all supported version of Python.

Reply:
While the change was introduced in 3.6 of the backport (as noted above), there have been various improvements to the new API up through version 3.10. Therefore we are requiring `importlib_metadata>=3.10`.

Reply:
By the way, I had intended to include an explanation in the commit message. So thanks for the reminder.